README.md

```diff
@@ -76,3 +76,26 @@ To compile MO files and update live translations
 $ django-admin.py compilemessages ;
 $ for app in {main,viewer} ; do cd kobocat/apps/${app} && django-admin.py compilemessages && cd - ; done
 ```
+## Testing in Kobocat
+
+Enter the container
+
+``` sh
+$ docker exec -it {{kobocat container}} /bin/bash
```

Comment:
You could add command with kobo-install too. (i.e. `For kobo-install users `./run.py -cf exec kobocat bash`)